LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -206,6 +209,13 @@ public LoRaDevice(string devAddr, string devEUI, ILoRaDeviceClientConnectionMana
             ClassType = LoRaDeviceClassType.A;
         }
 
+        /// <summary>
+        /// Use constructor for test code only.
+        /// </summary>
+        internal LoRaDevice(string devAddr, string devEUI, ILoRaDeviceClientConnectionManager connectionManager)
```

Comment:
This was added in order to avoid refactoring the tests to use the other constructor right? Seems fair to me.

Reply:
Exatly, otherwise we would have had a larger diff. We probably need to rethink this with #808